.chloggen/nil-empty-slice.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
```

```suggestion
change_type: 'breaking'
```

am not sure if the previous behavior is intentional, but this PR is a breaking change to confmap users